### DISCUSSION TITLE
ScriptReference/MenuCommand

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#310] [Notes]
# ScriptReference/MenuCommand

https://docs.unity3d.com/ScriptReference/MenuCommand.html

**Comment by SoylentGraham** [comment]
To handle multiple-selections (as MenuCommand.context is a single object), your MenuCommand function is called for every object selected.  
E.g. Selecting 10 game objects with RigidBody components, calls DoSomething() 10 times with different contexts.
